app/style/MiscComponents.less

```diff
@@ -239,6 +239,34 @@
   opacity: 0.7;
 }
 
+.path-button {
+  width: 16px;
+  height: 16px;
+  border: none;
+  outline: none;
+  background-color: transparent;
+  background-image: @search-icon;
+  background-size: 16px 16px;
+}
+
+.path-button:hover {
+  opacity: 0.7;
+}
+
+.path-input {
+  border-style: none;
+  cursor: text;
+  background: transparent;
+  outline: none;
+  align-items: center;
+  border-bottom: 1px @stroke-color-default solid;
+  font-family: @font-family-regular;
+  font-size: 13px;
+  color: white;
+  padding: 3px 4px 1px 10px;
+  width: 400px;
```

Comment:
Don't use a fixed width on the input here, let it fill the container component, as we may want to use it in different contexts with different widths.

If you want to show a component with the input+button (and if that's why you used the fixed width) then define a new component with both sub-components and style them such that the input leaves room for the button on the side.

Something to the effect of:

```jsx
const PathInputAndButton = (...) => (
  <div class="class-for-inline-pathinput+button">
    <PathInput ... />
    <PathInputButton .../>
  </div>
);
```

Reply:
once again, from the react-native land. I would have used a flex on the parent and divided it with the children. sass is a little new to me